pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -103,14 +113,63 @@ func (c *ConnectionRecorder) GetConnection(connectionID string) (*ConnectionReco
 	return &ConnectionRecord{State: string(name)}, nil
 }
 
-// invitationKey computes key for invitation object
-func invitationKey(verKey string) (string, error) {
-	storeKey, err := computeHash([]byte(verKey))
+// saveConnectionRecord saved the connection record against the connection id  in the store
+func (c *ConnectionRecorder) saveConnectionRecord(msg *message) error {
+	k, err := connectionKey(msg.record.ConnectionID)
```

Comment:
Can we just pass record here instead of "msg" ? For both key and value record is used. key - record.Connection and value - record.

This way we don't want [this](https://github.com/hyperledger/aries-framework-go/pull/443/files#diff-f761c91e753eba1c42a68076df06ce12R61) change as well.

Reply:
done